.github/workflows/build+release.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and upload release
+
+on:
+    workflow_dispatch: # trigger manually
+        inputs:
+            tag_override:
+                description: "Tag"
+                type: string
+                required: false
+            release_notes:
+                description: "Release notes"
+                type: string
+                required: true
+
+env:
+    node_version: "16.x"
+
+jobs:
+    build-and-upload-release:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Apply tag override and parse ref
+              uses: bisgardo/github-action-parse-ref@v1
+              id: ref
+              with:
+                  ref: "${{github.event.inputs.tag_override || github.ref}}"
+                  default-ref-type: tags
+            - name: Fail if ref is not a tag
+              if: "steps.ref.outputs.ref-type != 'tags'"
+              run: exit 1
+            - name: Parse tag name
+              uses: bisgardo/github-action-regex-parse@v1
```

Comment:
I'm surprised you had to make your own regex parser for this? :thinking:

Reply:
I'm sorry, but why do we have such special requirements to have to maintain a custom tag parser for this?

Reply:
@shjortConcordium I actually did it as an exercise before I realized that could be useful here, so I honestly haven't looked that intensely. But a quick search indeed doesn't find anything better.